modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/TableTestUtils.java

```diff
@@ -26,10 +26,7 @@
 import org.apache.ignite.internal.catalog.commands.ColumnParams;
 import org.apache.ignite.internal.catalog.commands.CreateTableCommand;
 
-/**
- * Utils to manage tables inside tests.
- */
-// TODO: IGNITE-19502 - remove after switching to the Catalog.
+/** Utils to manage tables inside tests. */
```

Comment:
Let's remove the class altogether: it's not used anywhere (at least, on the `main`). I forgot to remove it when I did a cleanup.

Reply:
Get rid of